nova/src/folding/hypernova/nimfs.rs

```diff
@@ -0,0 +1,337 @@
+#![allow(unused)]
```

Comment:
There's a name clash as hypernova paper also calls the scheme "multifolding", because there're 2 different LCCS and CCS relations. I was thinking of renaming cyclefold to something else during the last refactoring, but left it as is.

Reply:
Yeah I handled this by just alias'ing the import into the cyclefold module to `HNProof`, which works well enough